packages/api-client/src/api/serializers/cart.ts

```diff
@@ -47,7 +47,7 @@ const deserializeLineItem = (lineItem: any, attachments: JsonApiDocument[], conf
     _description: '',
     _categoriesRef: [],
     name: lineItem.attributes.name,
-    sku: '',
+    sku: lineItem.attributes.slug,
```

Comment:
Slug is not SKU, so let's not assign it.
Maybe we should assign slug to another attribute (`slug` or `productSlug`) and just extract that attribute when linking to product page?

Reply:
I agree, took care of that. Should be good now